lib/gitgitgadget.ts

```diff
@@ -271,35 +271,12 @@ export class GitGitGadget {
         // update work repo from base
         await this.updateNotesAndPullRef(pr.baseOwner, pr.number, previousTag);
 
-        // Remove template from description (if template exists)
-        let prBody: string;
-        try {
-            let prTemplate =
-                await git(["show",
-                           "upstream/master:.github/PULL_REQUEST_TEMPLATE.md"],
-                          { workDir: this.workDir });
-            // github uses \r\n so make sure it is set
-            prTemplate = prTemplate.replace(/\r?\n/g, "\r\n");
-            prBody = pr.body.replace(prTemplate, "");
-        } catch {
-            prBody = pr.body;
-        }
-
-        if (!prBody.length) {       // reject empty description
-            throw new Error("A pull request description must be provided");
-        }
-
-        // if known, add submitter to email chain
-        const ccSubmitter = userInfo.email ? `\nCc: ${
-            userInfo.name} <${userInfo.email}>` : "";
-        const description = `${pr.title}\n\n${prBody}${ccSubmitter}`;
-
         const series =
             await PatchSeries.getFromNotes(this.notes, pr.pullRequestURL,
-                                           description, pr.baseLabel,
+                                           pr.title, pr.body, pr.baseLabel,
```

Comment:
Should the `pr` and `userInfo` be passed to simplify the parm list?  This is a lot of parameters.

Reply:
Thanks for your review! Unfortunately I need to switch to my git PR at the moment, and hopefully I will get back to this PR and polish it in a couple days.

Reply:
Sorry for not replying when you mentioned it for the first time.

Yes, this sounds reasonable, but even before my patch, this was already the case. I thought that @dscho had some reasons for it. Maybe this is some kind of abstraction problem, or he wanted to make testing easier? I don't know. So I decided to just follow the same path.